packages/evolution-frontend/tests/ui-testing/testHelpers.ts

```diff
@@ -628,3 +689,44 @@ export const waitTextVisible = ({ context, text, isVisible = true }: { text: Pat
         }
     });
 };
+
+const continueOnInvalidPage: ContinueWithInvalidEntriesTest = async ({ context, text, currentPageUrl, nextPageUrl }) => {
+    const button = context.page.getByRole('button', { name: text });
+    await button.scrollIntoViewIfNeeded();
+    await button.click();
+    await expect(context.page.waitForURL(nextPageUrl, {timeout: 1000})).rejects.toThrow("Timeout")
```

Comment:
It's a clever wait to kind of add a timeout before going to a next page. I wonder if we would have any other mechanism to realize that we did not go to the next page, like waiting for any widget with the invalid class?

Reply:
We could check that a specific widget gets an invalid class on top of us remaining on the page. The function I wrote is a more general one where we try to advance without filling anything, but we could also do something where we test a specific widget.